backend/inspirehep/disambiguation/tasks.py

```diff
@@ -70,26 +75,127 @@ def disambiguate_signatures(self, clusters):
     db.session.commit()
 
 
+def match_literature_author_with_config(author_data, matcher_config):
+    matched_records = [
+        matched_record
+        for matched_record in match(author_data, matcher_config)
+        if matched_record.get("inner_hits")
+    ]
+    matched_authors_refs = match_literature_author_ids(matched_records)
+    return matched_authors_refs, matched_records
+
+
+def match_literature_author_with_validator(validators, author_data, matched_references):
```

Comment:
this doesn't call the matcher (doesn't do any query), so maybe
```suggestion
def validate_literature_author_matches(validators, author_data, matched_references):
```